wp-js-plugin-starter.php

```diff
@@ -31,11 +31,12 @@ function wp_js_plugin_starter_register_block() {
 	wp_register_script(
 		'wp-js-plugin-starter',
 		wp_js_plugin_starter_url( 'dist/index.js' ),
-		array( 'wp-element' )
+		array( 'wp-blocks' ),
```

Comment:
Thanks for the update, I think we need both `wp-element` and `wp-blocks` here. Also as you may know, the commits in this particular repository have an importance. So I might have to rebase your commit post merge (which means you won't be visible as a contributor) :(